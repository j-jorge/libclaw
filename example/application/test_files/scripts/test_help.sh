#!/bin/sh

./ex-application -h
./ex-application --help
