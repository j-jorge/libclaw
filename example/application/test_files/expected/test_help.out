./ex-application [-b] [-h] [--log-file=file] [--log-level=string] [--string=any_string] more_string...

	-b	Sample boolean argument.
	-h, --help	Print this help screen.
	--log-file	The file to use to store log informations.
	--log-level	Level of log informations:
		error: error messages,
		warning: warning and error messages,
		verbose: all messages.
	--string	Print a string.
Testing error_level:
Testing warning_level:
Testing verbose_level:
'-b' argument is missing.
'--string' argument value is ''
./ex-application [-b] [-h] [--log-file=file] [--log-level=string] [--string=any_string] more_string...

	-b	Sample boolean argument.
	-h, --help	Print this help screen.
	--log-file	The file to use to store log informations.
	--log-level	Level of log informations:
		error: error messages,
		warning: warning and error messages,
		verbose: all messages.
	--string	Print a string.
Testing error_level:
Testing warning_level:
Testing verbose_level:
'-b' argument is missing.
'--string' argument value is ''
